reaktive/src/commonMain/kotlin/com/badoo/reaktive/completable/DoOnAfter.kt

```diff
@@ -11,6 +11,10 @@ import com.badoo.reaktive.disposable.SerialDisposable
 import com.badoo.reaktive.disposable.doIfNotDisposed
 import com.badoo.reaktive.disposable.plusAssign
 
+/**
+ * Calls the shared `action` for each new observer with the [Disposable] sent to the downstream.
```

Comment:
You can use `[action]` here to reference method parameter. Even auto-completion works. Also it add some additional fancy stuff like click to check type.

![Screenshot 2021-06-11 at 12 32 04](https://user-images.githubusercontent.com/9081555/121680217-31e44e00-cab1-11eb-9fc5-b375cff20504.png)

![Screenshot 2021-06-11 at 12 34 05](https://user-images.githubusercontent.com/9081555/121680376-635d1980-cab1-11eb-970b-c667bc57af27.png)

![Screenshot 2021-06-11 at 12 32 28](https://user-images.githubusercontent.com/9081555/121680238-37da2f00-cab1-11eb-8996-d7854035f05a.png)

Reply:
Wow, didn't know that. Will do, thanks!

Reply:
Done